xbmc/utils/PerformanceSample.cpp

```diff
@@ -28,7 +28,7 @@
 #endif
 
 #ifdef HAS_PERFORMANCE_SAMPLE
-#include "Application.h"
```

Comment:
a define HAS_PERFORMANCE_SAMPLE in PerformanceSample.cpp makes not much sense in the times of cmake.